.aicoe-ci.yaml

```diff
@@ -0,0 +1,15 @@
+check:
```

Comment:
I wonder how they come into the game if we run the tekton pipelines via prow jobs? ;)